daemon/state.go

```diff
@@ -2,26 +2,16 @@ package daemon
 
 import (
 	"fmt"
-	"sync"
 	"time"
 
 	"github.com/docker/docker/daemon/execdriver"
 	"github.com/docker/docker/pkg/units"
+	"github.com/docker/docker/runconfig"
 )
 
 type State struct {
-	sync.Mutex
-	Running           bool
```

Comment:
With the new idea  to move types to api/types (so that they can be reused by various API) state should also be moved there? I agree mutex should remain in the daemon package.